lib/scholar/metrics/neighbors.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Scholar.Metrics.Neighbors do
```

Comment:
Neighbors or graph?

Reply:
Or rather: what is the shape of the inputs expected here? Is it expecting a graph shape? If so, how is the graph laid out? Or can we can pass it the result of any Neighbors classification?

Reply:
hmmm I would set it to neighbors as it can work with the queries as well, not only graph. I guess I should rename the arguments to something else.

Reply:
> Or rather: what is the shape of the inputs expected here? Is it expecting a graph shape? If so, how is the graph laid out? Or can we can pass it the result of any Neighbors classification?

The shape of both arguments is `{n, k}`. The function calculates how many nearest neighbors out of `k` possible are predicted correctly, then averages over all `n` points.
It doesn't work with classification, so maybe changing the module name to Graph or something else might a good idea.

Reply:
Or keep it as Neighbors as we have a separate module for metrics used for classification and regression.

Reply:
Yeah, agreed on Neighbors. We just need to improve the docs with more concrete examples!

Reply:
Yes, of course. I just submitted a PR draft to show why I didn't include k-NN search inside RandomProjectionForest module.